### PR TITLE
orphan the agent namespace when deployed by addontemplate

### PR DESCRIPTION
**Pull request by zhujian7**

### CODE REVIEW

File: charts/managed-serviceaccount/templates/addontemplate.yaml [hunk 1]
```diff
@@ -8,6 +8,12 @@ spec:
   agentSpec:
     workload:
       manifests:
+      - apiVersion: v1
```
Comment (qiujian16) [inline charts/managed-serviceaccount/templates/addontemplate.yaml:11]: add a comment here indicating this is to handle upgrade, and should be removed in the later release

Comment (zhujian7) [reply charts/managed-serviceaccount/templates/addontemplate.yaml:13]: PTAL

**Pull request merged**